tiledb/sm/cpp_api/config.h

```diff
@@ -878,6 +878,12 @@ class Config {
    * - `config.logging_format` <br>
    *    The logging format configured (DEFAULT or JSON)
    *    **Default**: "DEFAULT"
+   * - `profile_name` <br>
+   *    The name of the RestProfile to be used for REST transactions. <br>
+   *    **Default**: "default"
+   * - `profile_homedir` <br>
+   *    The local $HOME directory where the user profiles are saved. <br>
+   *    **Default**: tiledb::common::filesystem::home_directory()
```

Comment:
On the API PR now this is an argument passed to the rest profile API. I want to check the interaction between this and that.

On the API PR if the user supplies `nullptr` for the directory argument then it substitutes `$HOME`.  Is the intent instead to substitute this config directory?

Reply:
Not relevant anymore - I believe.